packages/core/src/types/toposort-class.d.ts

```diff
@@ -0,0 +1,6 @@
+declare module 'toposort-class' {
```

Comment:
We should add this to https://github.com/sequelize/toposort instead

Reply:
Let's add it here first and we can remove it in a future PR when we update toposort

Reply:
We can't because that's a global declaration. It's going to impact other users of that library that have sequelize installed

Reply:
Are you saying I should leave it as is or change it to sequelize/toposort?

Reply:
This file should not be added here because it would impact users in an unintended way. We will add it to sequelize/toposort later when we work again on that. For now we can ignore ts errors in the model manager I think

Reply:
done